index.rst

```diff
@@ -45,12 +42,53 @@ All of this data needs to be associated with this configuration in a way that is
 In this tech-note we discuss the basic organization of configurations and the operational procedures involved in handling CSC configuration.
 The document begins with the most simple case of bringing a CSC up with a specific configuration, and eventually details how to updating and testing newly defined configurations.
 
-Most CSCs handle configuration in the same manner, however, there are a few exception to the rule, notably the Camera CSCs.
-These special cases are addressed in section-appendix-configuration-non-salObj_.
 
-For more technical details about how CSCs handle configuration, including the definition of configuration related topics published by the CSC, see `tstn-017 <https://tstn-017.lsst.io>`__.
+Configuration Interface Summary
+===============================
+
+The standard interface for CSCs is defined in :lse:`209` and is the official reference/requirements document.
+This section provides a very brief overview of the key topics related to configuration that are continually mentioned in the sections below.
```

Comment:
I suggest dropping "that are continually mentioned in the sections below". It is implied.

Or at least change "continually mentioned" to something more like "described in more detail".